src/sentry/lang/javascript/processor.py

```diff
@@ -663,7 +673,7 @@ def expand_frames(self, frames, release):
                     else:
                         all_errors.append({
                             'type': EventError.JS_MISSING_SOURCE,
-                            'url': force_bytes(abs_path, errors='replace'),
+                            'url': expose_url(abs_path),
```

Comment:
Seems only a matter of time that someone will create an error without using `expose_url` ... what if we funneled through a helper function?